.github/ISSUE_TEMPLATE/new_module.yml

```diff
@@ -0,0 +1,36 @@
+name: New module
+description: Suggest a new module for nf-core/modules
+title: "new module: TOOL/SUBTOOL"
+labels: new module
```

Comment:
maybe we can also have a section here about information about the module. I.e. website, conda package etc..

Reply:
as optional information